vm/src/obj/objbytes.rs

```diff
@@ -269,6 +268,28 @@ impl PyBytesRef {
         obj => {Err(vm.new_type_error(format!("{} cannot be interpreted as an integer", obj)))}
         )
     }
+
+    #[pymethod(name = "count")]
+    fn count(
+        self,
+        sub: PyObjectRef,
+        start: OptionalArg<PyObjectRef>,
+        end: OptionalArg<PyObjectRef>,
+        vm: &VirtualMachine,
+    ) -> PyResult {
+        match is_bytes_like(&sub) {
+            Some(value) => Ok(vm.new_int(self.inner.count(value, start, end, vm)?)),
+            None => match_class!(sub,
+                i @ PyInt => {
+                    Ok(vm.new_int(self.inner.count(vec![i.as_bigint().is_byte(vm)?], start, end, vm)?))},
+                obj => {Err(vm.new_type_error(format!("argument should be integer or bytes-like object, not {}", obj)))}),
+        }
+    }
+
+    #[pymethod(name = "join")]
+    fn join(self, iter: PyIterable, vm: &VirtualMachine) -> PyResult {
```

Comment:
Maybe also add this method to bytearray.

Reply:
for now i'd like just focus on bytes method. Bytearray will need after a big cleanup